ui-tests/conf/wdio.conf.js

```diff
@@ -25,24 +25,24 @@ exports.config = {
         maxInstances: 5,
         browserName: 'chrome',
 
-    },
+        // },
 
 
-    {
-        maxInstances: 1,
-        browserName: 'MicrosoftEdge'
+        // {
+        //     maxInstances: 1,
+        //     browserName: 'MicrosoftEdge'
 
-    },
+        // },
 
-    {
-        maxInstances: 1,
-        browserName: 'firefox'
+        // {
+        //     maxInstances: 1,
+        //     browserName: 'firefox'
 
-    },
+        // },
 
-    {
-        maxInstances: 1,
-        browserName: 'safari',
+        // {
```

Comment:
I am not sure what this config does or why it's no longer needed, but usually better to delete than comment-out?

Reply:
you are right, will need to revert it, thank you